bundle/Controller/UserController.php

```diff
@@ -181,11 +182,25 @@ public function editAction(
             }
         }
 
+        try {
+            // assume main location if no location was provided
+            $location = $this->locationService->loadLocation((int)
+            $user->versionInfo->contentInfo->mainLocationId);
+        } catch (UnauthorizedException $e) {
+            // if no access to the main location assume content has multiple locations and first of them can be used
+            $availableLocations = $this->locationService->loadLocations($user->versionInfo->contentInfo);
+            $location = array_shift($availableLocations);
```

Comment:
I'm not entirely sure about that. AFAIK that Controller is used in Location view, so switching seamlessly to another location doesn't seem correct.